### DISCUSSION TITLE
Group all from one manager

### DISCUSSION THREAD

**Discussion by reccetech-ca** [renovatebot/renovate#23550] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.21.1

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Running renovate.json on a local repo with Jenkins plugins. Most of these plugins are interdependent so separate PRs will likely break dependencies in our tests. Ideally we run Renovate to create a single PR for all Jenkins updates, test, and merge together.

### What you tried so far.

Adding matchManagers and groupName got us part way there by grouping it into 2 PRs (one for major, one for minor/patch, branch=renovate/jenkins, and branch=renovate/major-jenkins).
Added matchUpdateTypes for patch, minor, major did not consolidate them
Current config:
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "enabledManagers": ["jenkins"],
  "labels": ["Renovate"],
  "packageRules": [
    {
      "matchManagers": ["jenkins"],
      "matchUpdateTypes": ["patch", "minor", "major"],
      "addLabels": ["Jenkins Plugin"],
      "groupName": "jenkins"
    }
  ]
}

Found similar questions but none of the solutions seemed applicable and I can't tell if I'm missing something in the config, or if matchManagers and matchUpdateTypes are incompatible when trying to group all into one PR via groupName

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
You want to mix major and non major updates together?

**Reply by viceice** [reply]
yes, almost all Jenkins plugins now only publish major versions

**Reply by rarkins** [reply, marked as answer]
separateMajorMinor=false for these dependencies then

**Reply by reccetech-ca** [reply]
Ah excellent I totally missed that flag as I assumed it was something in the packageRules.

**Reply by reccetech-ca** [reply]
Confirmed that fixed it and I removed the matchUpdateTypes as it was unnecessary for this case. Thanks for the help folks.

**Comment by viceice** [comment]
https://docs.renovatebot.com/configuration-options/#separatemajorminor
